### ISSUE TITLE
[BUG] sim_combined with multiple bursty oscillations

### ISSUE THREAD

**Issue by ryanhammonds** [neurodsp-tools/neurodsp#265]
I noticed this raises an error most of the time, but occasionally doesn't:

```python
from neurodsp.sim import sim_combined

components = {
    'sim_bursty_oscillation': [{'freq': 9},
                               {'freq': 10}],
}

n_seconds = 25
fs = 1000

sig = sim_combined(n_seconds, fs, components)
```

The error is raised in the `make_bursts` and the same code works replacing with `sim_oscillation`. My goal was to make a bursty + variable oscillation but there doesn't see to be any easy way to do this unless I'm missing something.

I will dig into the error more to figure out if a fix is needed.

**Comment by TomDonoghue** [comment]
I'm pretty sure (like 95%) that the underlying bug here is the same as #271, and addressed by #273 - at least, with the fix in 273, I don't get any errors from the above code. 

@ryanhammonds - when you get a chance, can you double check that 273 seems to fix this? If so, we can close this.

**Comment by TomDonoghue** [comment]
I'm pretty sure this is addressed by #273, so I'm going to close, but please re-open if this doesn't seem to be addressed!
